scripts/comparator/queries

```diff
@@ -0,0 +1,17 @@
+rate(quail[5m]):15s
```

Comment:
Ah I see, is it worth perhaps just making this file either JSON or YAML and doing a JSON or YAML parse?

Reply:
How about XML to keep it interesting? :p